exonum/src/helpers/fabric/builder.rs

```diff
@@ -162,15 +158,25 @@ impl NodeBuilder {
             .map(|mut factory| factory.make_service(ctx))
             .collect();
 
-        let consensus_passphrase =
-            ZeroizeOnDrop(env::var(EXONUM_CONSENSUS_PASS).unwrap_or_default());
-        let service_passphrase = ZeroizeOnDrop(env::var(EXONUM_SERVICE_PASS).unwrap_or_default());
-
-        let config = config.read_secret_keys(
-            &config_file_path,
-            consensus_passphrase.0.as_bytes(),
-            service_passphrase.0.as_bytes(),
-        );
+        let config = {
+            let run_config = ctx.get(keys::RUN_CONFIG).unwrap();
+            let consensus_passphrase = run_config
+                .consensus_pass_method
+                .parse::<PassInputMethod>()
+                .expect("Incorrect passphrase input method for consensus key.")
+                .get_passphrase(SecretKeyType::Consensus, true);
```

Comment:
Overall I agree but not in this case since I do not think that this enum is going to be extended and this makes function call bigger. 
I renamed this argument to `node_run`, also this function has comment about this parameter.